core/src/main/resources/jenkins/model/experimentalflags/Messages.properties

```
@@ -0,0 +1,23 @@
+# The MIT License
+# 
+# Copyright (c) 2022, CloudBees, Inc.
+# 
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+# 
+# The above copyright notice and this permission notice shall be included in
+# all copies or substantial portions of the Software.
+# 
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+# THE SOFTWARE.
+
+UserExperimentalFlagsProperty.DisplayName=Experimental Flags
```

What do you think about just calling this Experiments? i.e. sounds like better user facing terminology to me
```suggestion
UserExperimentalFlagsProperty.DisplayName=Experiments
```

---

Moving with `UI Experiments`, is it OK?

_(I cannot find any example of a user feature flag that is not related to UI, as other features are more expected to use SystemProperty for the instance)_

---

why qualify it? seems weird, especially to abbreviate it, thinking of this as a user of Jenkins rather than a developer will they know what UI stands for?

---

I would think that the qualification help about what it is about.

"User Interface Experiments"?

No strong opinion, just thinking that the "Experiments" could not necessarily convey the meaning.

---

I wouldn't add too much when we don't really know for sure how it will be used, given there's no PR that's jumping to use this currently.

If you must then I think `User Interface Experiments` is better, but I prefer it without the qualification